### DISCUSSION TITLE
articles/the-history-of-neural-networks

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#189] [Comments]
# articles/the-history-of-neural-networks

TL;DR 本文8200+字，全文阅读约需15分钟。本文简要回顾了从感知机到深度学习最新大模型应用 ChatGPT 的历史。

https://hutusi.com/articles/the-history-of-neural-networks

**Comment by Ralvine** [comment]
cool 很清晰通俗的一篇文章

**Comment by vczyh** [comment]
谢谢科普

**Comment by elonlo** [comment]
写的通俗易懂，赞！

**Comment by fireflyddt** [comment]
很好的文章

**Comment by wangdefou-dev** [comment]
感谢科普，居然看懂了一些

**Comment by szy0syz** [comment]
条理奇清，好文！❻❻❻

**Comment by Nagi-ovo** [comment]
读起来很爽，一气呵成，感谢科普！

**Comment by soarhone** [comment]
兴趣使然，最近在从零开始学习Python和人工智能。然后瞌睡遇到枕头，发现了这篇文章，拜读后我由衷地感叹，真的是一篇难得的好文!  非常感谢作者的付出，让我对神经网络有了一个基础的认知。我希望我也能循着作者的阅读路径把那些书看完，算是给自己今年的一个小目标。

**Reply by hutusi** [reply]
共同努力学习：）

**Reply by inceptiongt** [reply]
同感。从管中窥豹到一目了然，收获颇丰 :+1:

**Reply by sundev126** [reply]
一样，对深度学习有了大概的了解！感谢作者

**Comment by ycjcl868** [comment]
感谢作者，真的是一文讲清楚了 AI 发展的来龙去脉！！！

**Comment by Yunz93** [comment]
赞，通俗易懂

**Comment by liquid207** [comment]
感谢科普，懂了一些。这些概念不那么神秘了，大力出奇迹。

**Comment by inceptiongt** [comment]
内容看了个开头，希望在页面上加个目录。

**Reply by inceptiongt** [reply]
看完了，发现文章只有几个一级标题，并不需要目录 :joy:

**Reply by hutusi** [reply]
哈哈，谢谢~~

**Comment by Joker3Chen** [comment]
好文！谢谢作者的分享

**Comment by yexia553** [comment]
很好的科普，对非AI从业人员很友好，感谢

**Reply by hutusi** [reply]
大家的认可对我来说是很大的鼓励，谢谢各位~~

**Comment by xiepeng21** [comment]
非常好的科普文章，逻辑清晰，行文流畅：）

**Comment by AlexVagrant** [comment]
感谢作者，清晰好文

**Comment by zwz1437** [comment]
很棒的文章，可以转载到自己的博客吗？无商业用途

**Reply by hutusi** [reply]
可以的，请注明出处和原文链接，谢谢~~

**Comment by juglans** [comment]
作者真是用心了

**Reply by juglans** [reply]
非常好的一篇 [[AI]]科普文章[从神经网络到 Hugging Face](https://hutusi.com/articles/the-history-of-neural-networks)，作者阅读了大量书籍和资料后，从AI的源起和发展，到[[深度学习]]的主要流派技术特点，到 [[Hugging Face]] 社区的发展等等。
